docs/content/DevelopmentPlan/DevelopmentPlan.md

```diff
@@ -7,141 +7,333 @@ date: "2024-09-16"
 
-How often will you meet with your industry advisor? when? where?
+Meeting chairs and notetakes will be responsible for ensuring that given issues are filled out.
+For meeting with supervisors, notes should be taken under [[meetings]].
 
-Will meetings be virtual? At least some meetings should likely be in-person.
+Roles assignment can be found under [[DevelopmentPlan/DevelopmentPlan#Team Member Roles|Team Member Roles]].
 
-How will the meetings be structured? There should be a chair for all meetings. There should be an agenda for all meetings.
+Additional meetings can also be held as needed (knowledge sharing, coworking, etc.)
 
 ## Team Communication Plan
 
-Issues on GitHub should be part of your communication plan.
+Hinterland members will use Teams for all asynchronous and synchronous communication.
+
+Additionally, GitHub [issue tracker](https://github.com/aarnphm/tinymorph/issues) should be used for feature
+development. The Sprint [Kaban](https://github.com/users/aarnphm/projects/4?query=sort:updated-desc+is:open) board will
+also be used concurrently for tracking tasks and in-progress work.
+
+Expectations among team members with regards response to issues and pull requests assignments should be resolved within
+1-2 business days. Should any issues arise that prevents members from using any of the aforementioned communication
+methods, Email should be used to communicate with the rest of the team and the instructor.
 
 ## Team Member Roles
 
-You should identify the types of roles you anticipate, like notetaker, leader, meeting chair, reviewer. Assigning specific people to those roles is not necessary at this stage. In a student team the role of the individuals will likely change throughout the year.
+The following encapsulates member roles for Hinterland.
+
+[Aaron Pham](https://aarnphm.xyz) will take the lead role for the team and acts as [BDFL](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life) for `tinymorph`.
+They will be responsible for overall project management, communication among course instructors, supervisor, as well as providing support to the rest of the team
+
+Notetakers will be responsible for taking notes during meetings, lectures during the working week, and this role will
+be rotated among team members on a four week basis. The following can be used:
+
+```mermaid
+flowchart LR
+A[Aaron Pham] --> B[Waleed Malik] --> C[Nebras Khan] --> D[Lucas Li]
+```
+
+> [!note] Meeting chair assignment
+> Follow the schedule for notetakers.
+
+Given that `tinymorph` will include two components (web-based editor and ML inference server), the following include
+a rough draft of each champion's responsibilities:
+
+- design work: The team will collaborate on general design works for interfaces, lead by Aaron Pham.
+- web-based editor: [Nebras Khan](https://github.com/nebrask).
+- ML inference server: [Waleed Malik](https://github.com/waleedmalik7)
+- ML attention intervention: Aaron Pham
```

Comment:
1) we should probably link aarons github to this too ([Aaron Pham](https://github.com/https://github.com/aarnphm))

2) I believe Lucas will also be working alongside on the ML components

Reply:
updated